package.json

```diff
@@ -63,6 +63,8 @@
     "redux-saga": "0.12.0",
     "source-map-support": "ncalexan/node-source-map-support#fileUrls-plus",
     "tmp": "0.0.29",
+    "transit-immutable-js": "0.6.0",
```

Comment:
We can definitely naively serialize the entire app state as json. After all, immutable objects are convertible to plain js objects. And indeed the json output from `transit` is very difficult to read by humans.

That being said, not doing so means writing custom deserializers from json to records, since we can't just easily use built-in immutable methods when converting back. To elaborate, creating an immutable.js objects from json isn't guaranteed to be the exact same as the original serialized immutable.js object. This is because immutable.js has higher level data structure types (such as sets and lists which convert to plain arrays in json), so information about the data types is lost when converting to json.

In short, if we don't use `transit` for serialization, we'll have to implement the same functionality ourselves if we want to restore the entire app state. If not, we abandon the entire idea and don't dump the app state to disk to begin with, but some 'view' of it.